src/styles/breeding.less

```diff
@@ -68,6 +67,27 @@
     opacity: .5;
     pointer-events: none;
   }
+  .pokemonName {
```

Comment:
These class names are a bit generic. We have pokemon names all over the place, but this only affects one of the places

Reply:
Made the class names more specific to their actual use-case.